Bottom: 65426338b4ba4a2fa1420376e9d24ce700cad3d4
Top:    04c1c23d78a41e29dbc4066df226425715ba2f69
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 20:42:04 +0100

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index cfde6a9..3f442ce 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -213,7 +213,9 @@ typedef struct
   GdkDrawable *drawable;	/* Current drawable for drawing routines */
   gint width, height;
 
+#ifdef ENABLE_GL
   GdkGLConfig *glconfig;
+#endif
 
   gint trans_lines;
